_posts/2020-06-14-Emendations to Source Editions on IrishGen.md

```diff
@@ -0,0 +1,51 @@
+
+## The Edition
+The passage quoted above is [the entry on Cenél Lóegaire](https://celt.ucc.ie//published/G800011F/index.html) from the edition of the Book of Leinster as digitised on CELT (the translation is my own). In [the printed edition](https://www.vanhamel.nl/codecs/Best,_et_al._1954-1983) (p. 1467), however, it is made clear that passage contains a section that has been added in the margin of the manuscript page. I have emphasised the inserted material in bold; it is presented in a footnote in the printed edition but placed after "m Fheradaig".
+
+> Cu Ulad m Domnaill m Gillai Ultain m Oengusa m Cainelbain m Mael Chróin m Domnaill m Con Rai m Oengusa m Fheradaig **Araide Bibre in cáinte de Mumnechaib ba sé ba rectaire do Chormac .h. Chuind. Cairech a ben. Is i ro anacht Fiacha mc Oengusa. inde dicitur Fiacha Araide a quo Dal Araide.** m Mael Duin Dergainig m Colmáin m Aeda m Libuir m Dalleni m Ennai m Loegaire m Neill Noigiallaig.
+> > Cú Ulad, son of Domnall, son of Gilla Ultain, son of Óengus, son of Caindelbán, son of Mael Chróin, son of Domnall, son of Cú Rai, son of Óengus, son of Feradach **Araide Bibre (The satirist from among the Munstermen. He was steward to Cormac, grandson of Conn. Cairech was his wife. It was she who cared for Fiacha, son of Óengus. Thence he is called Fiacha Araide, whence are called the Dál nAraide)**, son of Máel Duin Dergainig, son of Colmán, son of Áed, son of Libur, son of Dallene, son of Enna, son of Lóegaire, son of Níall Noígiallach.
+
+## The Manuscript
+Turning to the manuscript, the insertion (now difficult to read) appears in the lower margin, with a special symbol (an upward pointing angle containing three dots) that will mark its point of insertion in the main text:
+
+!["blogpost2_img#2"](insertion_point.jpg)
+*Detail from The Book of Leinster (TCD MS 1339), p.335, lower margin. Source: [ISOS](https://www.isos.dias.ie)*
+
+Within the pedigrees, this symbol is indeed placed next to "m Fheradaig" in the Cenél Lóegaire pedigree:
+
+!["blogpost2_img#1"](insertion_point.jpg)
+*Detail from The Book of Leinster (TCD MS 1339), p.335, cols d-h. Source: [ISOS](https://www.isos.dias.ie)*
+
+However, two columns to the right of "m Fheradaig" is [a Dál nAraide royal pedigree](https://celt.ucc.ie//published/G800011F/text063.html) and, pretty much aligned on the page with "m Fheradaig" is Fiacha Araide, the subject of the insertion. This makes for a much more cogent place for the insertion, in terms of both chronology and subject-matter. Scribal error seems to be behind the association of the insertion with Cenél Lóegaire, as there is plenty of space next to "m Fhiachach Araide" for the insertion mark, so there was no need to place it so far away.
+
+## Course of Action on IrishGen
+
+We have therefore encoded the insertion as part of the Dál nAraide pedigree (http://example.com/LL/dail_araide.trig), which we read as follows (insertion in bold):
```

Comment:
We may want to use URLs to the repository while we do not have a website to point to.

Reply:
Ok. What do those look like?

Reply:
Like this `https://github.com/cyocum/irish-gen/blob/master/LL/dail_araide.trig`.  You can click through the Github webpage and bring up the entire file.  That URL for the file should always exist.

Reply:
I did not know that! Yes, we should definitely use those.